package.json

```diff
@@ -3,30 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@testing-library/jest-dom": "^5.11.10",
-    "@testing-library/react": "^11.2.6",
-    "@testing-library/user-event": "^12.8.3",
-    "@types/jest": "^26.0.22",
-    "@types/node": "^12.20.7",
-    "@types/react": "^17.0.3",
-    "@types/react-dom": "^17.0.3",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "@testing-library/jest-dom": "5.11.10",
+    "@testing-library/react": "11.2.6",
+    "@testing-library/user-event": "12.8.3",
+    "@types/jest": "26.0.22",
+    "@types/node": "12.20.7",
+    "@types/react": "17.0.3",
+    "@types/react-dom": "17.0.3",
+    "react": "17.0.2",
+    "react-dom": "17.0.2",
     "react-scripts": "4.0.3",
-    "typescript": "^4.2.3",
-    "web-vitals": "^1.1.1"
+    "web-vitals": "1.1.1"
   },
```

Comment:
Please add `typescript `to devDependencies

![image](https://user-images.githubusercontent.com/18407086/114225870-8b2de500-997b-11eb-88e8-1b53c66e9d2a.png)